config/deploy.rb

```diff
@@ -5,10 +5,10 @@
 
 set :application, 'pulmap'
 set :repo_url, 'https://github.com/pulibrary/pulmap.git'
-set :branch, 'master'
+set :branch, 'main'
```

Comment:
Can we delete this line altogether since we have the one below?

Reply:
Good catch. Fixed.